platform-service/sample.env

```diff
@@ -31,6 +31,7 @@ FLIPT_SERVICE_AVAILABLE=False
 # Cost calculation related ENVs
 MODEL_PRICES_URL="https://raw.githubusercontent.com/BerriAI/litellm/main/model_prices_and_context_window.json"
 MODEL_PRICES_TTL_IN_DAYS=7
-MODEL_PRICES_FILE_PATH="/tmp/model_prices.json"
+MODEL_PRICES_FILE_PATH="fsspec-test/cost/model_prices.json"
```

Comment:
@gaya3-zipstack Sample value can just be something without references to fsspec to indicate that it's a path to `model_prices.json`.

Reply:
Done